src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -356,5 +359,20 @@ Polymer({
     const port = Number(value);
     const valid = !Number.isNaN(port) && port >= 1 && port <= 65535 && Number.isInteger(port);
     return valid ? '' : this.localize('error-keys-port-bad-input');
+  },
+
+  _getInternationalizedUnit(bytesAmount, language) {
+    return formatBytesParts(bytesAmount, language).unit;
+  },
+
+  _getTranslatedDate(language, date) {
+    // We can't use a default Date object -- it still shows up as null, I'm not sure the lifetime
+    // properties when you pass an object into Polymer.
```

Comment:
I wonder if this happens because we set the default value in the Polymer property configuration. I think this is one of the only files that sets defaults that way.

Does it work if you set `serverCreationDate` in the constructor?

Reply:
I would guess that it does, but I don't want to change this file to use a constructor in this PR, it's too much going on already and it's not important enough to keep delaying the per key data limits ui for